RELEASING.md

```diff
@@ -15,6 +15,7 @@ We pre-release the provider to make for testing purpose. **A Pre-release is not
 - Using our [Release GitHub Action](https://github.com/mongodb/terraform-provider-mongodbatlas/actions/workflows/release.yml) run a new workflow using `master` and the following inputs:
   - Version number: vX.Y.Z-pre
   - Skip QA acceptance tests: Should be left empty. Only used in case failing tests have been encountered in QA and the team agrees the release can still de done, or successful run of QA acceptance tests has already been done with the most recent changes.
+  - Using an existing tag: Should be left empty (default `false` creates a new tag from `master`). This should be set to `true` only if you want to re-use an existing tag for the release and the tag name should adhere to our version number format.
```

Comment:
should we add in the comment that this will typically be useful when the release process failed (e.g. because tests failed) and want to try to release again

Reply:
I don't think that's necessary tbh as long as the tag exists for any use-case this can be used anyway